livechat/configuration/client.py

```diff
@@ -977,3 +978,9 @@ def get_license_webhooks_state(self,
 
 class ConfigurationApi33(ConfigurationApiInterface):
     ''' Configuration API client in version 3.3 class. '''
+    pass
```

Comment:
pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

Reply:
👍